client/landing/gutenboarding/index.tsx

```diff
@@ -2,37 +2,115 @@
  * External dependencies
  */
 import '@automattic/calypso-polyfills';
+import { setLocaleData } from '@wordpress/i18n';
+import { I18nProvider } from '@automattic/react-i18n';
+import { getLanguageFile } from '../../lib/i18n-utils/switch-locale';
 import React from 'react';
 import ReactDom from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 import config from '../../config';
+import { subscribe, select } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 import { Gutenboard } from './gutenboard';
 import { setupWpDataDebug } from './devtools';
 import accessibleFocus from 'lib/accessible-focus';
+import { USER_STORE } from './stores/user';
+
 /**
  * Style dependencies
  */
 import 'assets/stylesheets/gutenboarding.scss';
 import 'components/environment-badge/style.scss';
 
-window.AppBoot = () => {
+const DEFAULT_LOCALE_SLUG: string = config( 'i18n_default_locale_slug' );
+
+type User = import('@automattic/data-stores').User.CurrentUser;
```

Comment:
@jsnajdr Prettier removes space in the import parens here:

```suggestion
type User = import( '@automattic/data-stores' ).User.CurrentUser;
```